repobee_sanitizer/_syntax.py

```diff
@@ -66,6 +66,9 @@ def check_syntax(lines: List[str]) -> None:
     prefix = ""
     has_blocks = False
 
+    if _contained_marker(lines[0]) == Markers.SHRED:
+        has_blocks = True
+
```

Comment:
This is the fix to the bug I mentioned me i figured out as i was entering my sleep